EVFILT_USER is not implemented in OpenBSD kevent(2); not used in mattermost

Index: vendor/github.com/mailru/easygo/netpoll/kqueue.go
--- vendor/github.com/mailru/easygo/netpoll/kqueue.go.orig
+++ vendor/github.com/mailru/easygo/netpoll/kqueue.go
@@ -30,8 +30,8 @@ func (filter KeventFilter) String() (str string) {
 		return "EVFILT_SIGNAL"
 	case EVFILT_TIMER:
 		return "EVFILT_TIMER"
-	case EVFILT_USER:
-		return "EVFILT_USER"
+	//case EVFILT_USER:
+	//	return "EVFILT_USER"
 	case _EVFILT_CLOSED:
 		return "_EVFILT_CLOSED"
 	default:
@@ -100,7 +100,7 @@ const (
 	// EVFILT_USER establishes a user event identified by ident which is not
 	// associated with any kernel mechanism but is trig- gered by user level
 	// code.
-	EVFILT_USER = unix.EVFILT_USER
+	//EVFILT_USER = unix.EVFILT_USER
 
 	// Custom filter value signaling that kqueue instance get closed.
 	_EVFILT_CLOSED = -0x7f
